### DISCUSSION TITLE
new release(s)?

### DISCUSSION THREAD

**Discussion by jaromir-benes** [mkdocs/mkdocs#2351] [General]
Hi guys - Seeing continuous development on the github repo, I just wonder if you plan new releases? Or is it safe to upgrade from master any time? Best, Jaromir

**Comment by waylan** [comment]
The next release (1.2) will be made when all of the items in the [milestone][1] for that release are completed.

Regarding the master branch, you are at your won risk. As we are currently in the middle of a feature release, things are more volatile than they may be between bug-fix releases. That said, we require all tests to pass for each PR before we merge it. And well,  if you are using master, then we get someone who is testing new features before they get included in the release.

[1]: https://github.com/mkdocs/mkdocs/milestone/12
